src/react-apps/applications/receipt/.babelrc

```diff
@@ -0,0 +1,9 @@
+{
+    "presets": [
+        "env",
+        "react"
+    ],
+    "plugins": [
+        "react-hot-loader/babel"
+    ]
+}
```

Comment:
This configuration seems to be using old babel. Should be updated in a separate task.